instrumentation/rocketmq-client-4.8/library/src/main/java/io/opentelemetry/instrumentation/rocketmq/RocketMqProducerAttributeGetter.java

```diff
@@ -80,4 +81,20 @@ public String messageId(SendMessageContext request, @Nullable Void unused) {
     SendResult sendResult = request.getSendResult();
     return sendResult == null ? null : sendResult.getMsgId();
   }
+
+  @Nullable
+  @Override
+  public String messagePayload(SendMessageContext sendMessageContext) {
+    Message message = sendMessageContext.getMessage();
+    if (message == null) {
+      return null;
+    }
+
+    byte[] body = message.getBody();
+    if (body == null) {
+      return null;
+    }
+
+    return new String(body, StandardCharsets.UTF_8);
+  }
```

Comment:
Consider settings a `public static final long HS_MAX_PAYLOAD_SIZE = 65536` somewhere available and only collect the payload if it's under the threshold.

Reply:
done